src/errors/AuthFetchError.js

```diff
@@ -0,0 +1,9 @@
+class AuthFetchError extends Error {
+    constructor(message, response, body) {
+        super(message)
+        this.response = response
+        this.body = body
```

Comment:
Note this may not capture the stack trace correctly: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error#Custom_Error_Types

Reply:
Using `captureStackTrace` now. It's not part of `babel-plugin-transform-builtin-extend` out of the box. Should fly now. 🚀